api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -24,9 +24,24 @@ message KubernetesObjectData {
   oneof kind_specific {
     KubernetesPodData pod = 100;
     KubernetesNodeData node = 101;
+    KubernetesServiceData service = 102;
   }
 }
 
+message NodePorts {
```

Comment:
1. Move this message definition into the `KubernetesServiceData` definition below. I don't think this would ever need to be reused in another context.
1. Make this singular: ports -> port
1. As I suggested below, let's name this `Port` since it's containing info from the `spec.ports` field.